modules/docs/arrow-docs/docs/docs/arrow/data/mapk/README.md

```diff
@@ -11,13 +11,82 @@ redirect_from:
 {:.beginner}
 beginner
 
-### Supported type classes
+MapK is an Arrow wrapper over Kotlin Map type. The main goal is to make it a [type constructor](/docs/patterns/glossary/#type-constructors)
+and to work with Map in more functional way.
 
-```kotlin:ank:replace
-import arrow.reflect.*
+It can be created with calling `k()` function on Map
+
+```kotlin:ank
 import arrow.core.*
 
-DataType(MapK::class).tcMarkdownList()
```

Comment:
Keep these, they generate a list of typeclasses they implement! See the live docs: https://arrow-kt.io/docs/arrow/data/mapk/#supported-type-classes